examples/package.json

```diff
@@ -12,7 +12,10 @@
   },
   "scripts": {},
   "dependencies": {
-    "@clickhouse/client": "latest",
+    "@clickhouse/client": "latest"
+  },
+  "devDependencies": {
+    "split2": "^4.2.0",
```

Comment:
😄 did we missed it before?

Reply:
It could be - my local cache might have had it. Or maybe it comes from the dev dependencies of the client itself.